js/react/pages/profile/donation.jsx

```diff
@@ -31,11 +31,17 @@ function DonationHistory() {
 	useEffect(() => {
 		if (donations) {
 			setTotalGifts(donations.length);
+
+			let newTotalDonation = Number(0.0);
```

Comment:
I think the `Number` constructor can be removed since you already assign `0.0` as a value